apps/cli/src/base-program.ts

```diff
@@ -135,37 +136,56 @@ export abstract class BaseProgram {
 
   protected async exitIfLocked() {
     const userId = await this.exitIfNotAuthed();
-    if (await this.serviceContainer.cryptoService.hasUserKey()) {
+    let hasUserKey = false;
+
+    // If the process.env does not have a BW_SESSION key, then we will never be able to retrieve
+    // the auto user key from secure storage. This is because the auto user key is encrypted with
+    // the session key.
+    hasUserKey =
```

Comment:
I think after you refactored this we don't need to actually `let` this anymore.

```suggestion
    // If the process.env does not have a BW_SESSION key, then we will never be able to retrieve
    // the auto user key from secure storage. This is because the auto user key is encrypted with
    // the session key.
    const hasUserKey =
```

Reply:
Thank you for catching that! It's cleaned up!